ArduPlane/mode_auto.cpp

```diff
@@ -159,3 +159,29 @@ bool ModeAuto::_pre_arm_checks(size_t buflen, char *buffer) const
     // Note that this bypasses the base class checks
     return true;
 }
+
+bool ModeAuto::is_landing() const
+{
+    switch(plane.mission.get_current_nav_cmd().id) {
+    case MAV_CMD_NAV_LAND:
+    case MAV_CMD_NAV_VTOL_LAND:
+    case MAV_CMD_DO_VTOL_TRANSITION:
+        return true;
+    default:
```

Comment:
the default is not necessary, can you remove it?

Reply:
Sure thing! Working on a commit to remove this now.